clouddriver-artifacts/src/integration/resources/ssh/id_rsa_test

```diff
@@ -0,0 +1,50 @@
++Tb6UPmTt6vr6Yejo5nftJzN7Rp6GBnYq8IWZ29qNRxH8Xtqs3u8frwAIR9jAdhL/X9SUY
+PIqOIjmzNi6ilfAxsLLpo9329dFuyNRWyG6DqVUMX6pUynNpewWZbKPRw18UTrvQgG710n
+K6WFqE+2T8NOHaMsXw8qLnAhwctcDJZnT+4SBOqxi+BVeUBrLlSAZVwuKb9cUxdwSahOxI
+f3sFnbfPlgXohmtg99xMNyPv8My3dr9RvJWIo0LGlAFpwVM6t7l7dC1gmBSS0M2PHmr+V1
+/7l4i2RSI1+5IpHG0uCmqW/lRg0DuvuoY2UtbZYSd5BQAAB1BE3fOYir6MAAVLU3616LUP
+9QalNo4QNUjaywBaiDjbqOJywpXlHbvx1dVb1jETY0qmAzy6427OYJGXx2xlvGlNOgLw1K
+YkAmG1rVgF5o5mcamQ0wyf/Kqpff7a3T6bZsa8GNjYXtm3LTNpLzpHQ5Tk4XrpQOe67g7n
+DF+QOo2LZSZqrToiF1ptPkKncbMUmxsOv84HQkzcGFHFw549H+5d6qk7X5uDakqQqhdBnF
+eRyHi22v7KiXpp4NjOCSRw6e55eo+3y+Q7VUFxmRWBffMUuy4lloUKeUGwlu4lwsOfc4aB
+EdpJtMQZ978qg+DdT7cjuyTz2InPIOVcODvUfGXiq/xB8ZmnRZKT8Z9vIn1tcIB91Jbf5i
+pjXnjFNGPPrsZ7WwC/dhKtYY2nuAbyfiWrESlml9dRr32MNgAX6ScAkLC4kve3h4nndF6z
+3n6OzPPoD9pymBzSAzhxM8VEXensbP96n03VsP5633FgbIKRM3AJM0/7uoaHbFpXDWiBR9
+PRAPFu6WbLdREXBHtR5eRXTCyYr1V+PpEa1lrxqguClaQjERtMaB1Y3XDI6stFBpjJ6LK8
+Bt3tHrBjqhLFLsyShmSk8a0DEC5hKBMNQs2BzE67XMhsPW11CzfhaBuWz/xOFOViOrlVb8
+FDHjuNFlli9n3BrXdhSiCV3Ug+LcrSXSYVeB+GzzhKpve7Qgt5cm7U5NhS0qtl3fpvsOTF
+5YIZsn5Rp0TP/d1wxuWzIKPuCPkB3686aSu8y6CKv0k8PKN6IsY71MgxTHGs5POgUfhWLw
+iJEby/M2D4A1ckPYdzaMSJVNXepOk2izh2hQTgDS3v/meK6v0qd9ghzLcsW11ZDUESJhEk
+YNp5bfuaZTpWgyi7jMhNKP38KLhiHKxZBOTG720jFTFXRh1/+Ttbp4E0w7UA+PyLPoHHMp
+70VFeIftMOm8Wb/zxmDvyj/2XStKSl+nPwoCJJNiXPxW12envYS2PhKLMtWqqlCgBn3O3i
+gqrPj0fgNMP1C8subrn5JW01FmV8A651U93gSzVBMciF8TbeIw5WGavUWJAhU7lCrQDhGk
+iMayqThfpDLjvndTZwibfRL0Fe9D3bdS6s6uKMrDocWQ4QjW4iGEFf3+iL8ZsyxYc1m3Vb
+chBY9VXnQsD3V6lIl7+8KIn2mS5m1V2J/oS9unliWm3qM7apfJ2jXta1H/1lTPy2vB0jp6
+1Oe2cigboVbHz9PvGSHjsdlno4yKH8ESr+5rKDaP3MnkFyJB5B5TnBk7Z7phrVR2kax3FL
+IfLJgG2SOUi/fqb+MhlW9LHygxsdh7LbjjgoKDKwbBVuBA4oT2TN3zHGKdVecT7BmU1C96
+vAmh+pdmbO5jd0r8UHnEGZkFOKqjtM3CXuzSGORyT9cpPqyit/pMf/b7uYQ5mc/nzAKRfa
+KQbrIO5tBe1tyBeL43Q5nk+l7tcYKya789eBSSEog/TWb3Ip2563OW6ZLw0P1yLCaoLTUm
+jtBkEPdFvTiFnmLYzl9oXwd0jO29kcaCqJiPtqdexEaeUA/fj6m0QwUg1+37frbVEWMJRc
+mgjZyWJ+hhEsTBKkGokV/cx8/wXfO45LcH6WFoEzxRFVb0eMHd8k86H3p764jJ3tTeTnBE
+lmFeHqaWPDinjDBbmWc2w3yvl5JIv/aMU5BO9/tS87D1CkWS0IOKAn3uIVNqmeAnmYD8w1
+XzWymJgc67jv5Noyynqk0h3qGwo+BYviMQ5uscmJ1AiHls+ZGAdWOvQLL9iH9U8yu+q7Cs
++ZnhOy7FZ9/duKpOD/IG2/F0o+WjD/KDnCW69n6RcohYW83ESqI4Wl22MJXJFWhnPcUbUo
+OPh16mitCd0EQUAPdw/FdPs7kQ7CQlsTdFhKXFRkJpL/7x3EtyTvX9T9sS/h9xLbhIp3HJ
+TXNIhNJikjUoZ9/7ytiP/6js8g+nCHXl7eJnHSVbq+iYSWRR4qXk+l/+nmWhQPouVPQggp
+eCbaCDJh6S8mfZA+GHF5qhZcjYuwTaAtZOHxlRZM/TSHhJT6o2eiFIzXvqlaprwoJzWfQZ
+m1AI/feFuxw/Olceo3SzYLhKU1L2l+ZxMewPlN4J3RQrUkCWbM+qqSOZz6j4OsR09FDdyN
+StLeaBLymf2Kp+uReqXw5GkOqYMHOKxGxhGFEORZbKYKovLjiMex47spCn+DZvdX+LAbfG
+VVss+Zd2Ko+jTqaEhVCL0aIRMy2fY1NZPFG9SPbWKTQfpRUkPrhyM+HcfQr31+lcFuIldC
+wMyuBr4NzsIXuX2WLTjvIwI9ZogafPGFI+PZ+EiQEIB7+0wGkPmt7/vKiiyZDzbdVnnDy0
+J+SQEE43M1J+Fa7tfsONrURFzDjkpIjrisKayCt/q3Ye3f4gFZBlLAK3lcoEwZSFfmaqhd
+ghYiPpieX8QabfOBuopJRkMl+Di505j2BlXEd/46P2DyeaWTvz/y9vhRHjcxiyJOe9NzIA
+uf9VhW4rnuQtMsiPY1rTVX0TU=
+-----END OPENSSH PRIVATE KEY-----
```

Comment:
I'm probably just being nit-picky at this point so take this with a grain of salt - is this something that can be generated at test runtime? Something feels off about committing private SSH keys even if they are only for tests. I'm open to other opinions. @ncknt

Reply:
We already exec shell commands inside gitea docker container, so in theory this should also be possible.